devices.js

```diff
@@ -1069,6 +1069,16 @@ const devices = [
         fromZigbee: generic.light_onoff_brightness().fromZigbee,
         toZigbee: generic.light_onoff_brightness().toZigbee,
     },
+    {
+        zigbeeModel: ['RB 178 T'],
+        model: 'RB 178 T',
+        vendor: 'Innr',
+        description: 'E27 Bulb warm dimming',
+        supports: generic.light_onoff_brightness().supports,
+        fromZigbee: generic.light_onoff_brightness().fromZigbee,
+        toZigbee: generic.light_onoff_brightness().toZigbee,
```

Comment:
It seems that this device also supports color temperature, is that right? (https://www.amazon.co.uk/tunable-white-RB-178-pack/dp/B075QM4YLX)